app/src/main/java/com/todoist_android/data/network/APIAuthentication.kt

```diff
@@ -21,4 +24,12 @@ interface APIAuthentication {
         @Field("email") email: String,
         @Field("password") password: String
     ) : SignupResponse
+
+
+    @POST("/tasks")
+    suspend fun addTasks(@Body tasksRequest: AddTaskRequest): AddTasksResponse
+
+    @POST("tasks")
```

Comment:
`@POST("/tasks") //notice you didn't put the trailing hash `

Reply:
noticed that also, and made the change.
Once the recycler view is done I will push the most recent changes.

Reply:
For now since am using a sample todo model and not the end point, the edit functionality won't be affected.